tests/examples/shims/screen.lua

```diff
@@ -34,32 +34,27 @@ local function create_screen(args)
 
     local wa = args.workarea_sides or 10
 
-    -- This will happen if `clear()` is called
-    if mouse and not mouse.screen then
-        mouse.screen = s
-    end
```

Comment:
I wouldn't expect a commit with commit message "shims: Fix indentation" to remove four lines from a file....

Reply:
whoops, rebase conflict issue, it belongs to the other commit. That's now handled by the mouse metadable instead of the screen shim.

Reply:
done